### DISCUSSION TITLE
Azure dev ops

### DISCUSSION THREAD

**Discussion by craftzneko** [sveltia/sveltia-cms#2] [Q&A]
This is looking really nice.  Does it support an azure dev ops backend like netlifycms does here https://decapcms.org/docs/azure-backend/ out of the box or could it eventually? Cheers

**Comment by kyoshino** [comment]
Hey @craftzneko, thanks for your comment! I’ll check if Azure provides an API to fetch the Git repository contents in bulk. If not, I’m not going to add support for the specific service, unfortunately.

**Reply by craftzneko** [reply]
@kyoshino Is this useful https://learn.microsoft.com/en-us/rest/api/azure/devops/git/repositories/get-repository?view=azure-devops-rest-7.0&tabs=HTTP, really interested if this would usable for us. Great work btw on the project, it looks amazing!

**Comment by kyoshino** [comment]
[Items - Get Items Batch](https://learn.microsoft.com/en-us/rest/api/azure/devops/git/items/get-items-batch?view=azure-devops-rest-7.0&tabs=HTTP) is probably what I need, but unlike [Items - Get](https://learn.microsoft.com/en-us/rest/api/azure/devops/git/items/get?view=azure-devops-rest-7.0&tabs=HTTP) it doesn’t return file content. I’ll dig into further and also check how Azure is supported in Netlify CMS. Basically, I don’t want to send 100 requests to get 100 files, which leads to bad performance and UX 😏

**Comment by kyoshino** [comment]
I’ve checked the latest version of the Azure REST API docs but still can’t find a way to retrieve file content for multiple files. Also, I think the limitations mentioned in the [Decap CMS doc](https://decapcms.org/docs/azure-backend/) still exist. So, unfortunately, the Azure backend can’t be added at this time. I’ll keep checking the API docs to see if we can support the platform in the future.
